src/components/m-accordion/index.js

```diff
@@ -0,0 +1,76 @@
+import { BaseComponentGlobal } from '../_abstract/component-types';
+import getAttribute from '../../js/get-attribute';
+import on from '../../js/on';
+// import the styles used for this component
+import styles from './index.scss';
+// import the template used for this component
+import template from './_template';
+import wcdomready from '../../js/wcdomready';
+
+class AXAAccordion extends BaseComponentGlobal {
+  constructor() {
+    super(styles, template);
+  }
+
+  /**
+   * REF: https://www.w3.org/TR/custom-elements/#custom-element-conformance
+   */
+  connectedCallback() {
+    super.connectedCallback();
+
+    this.className = `${this.initialClassName} m-accordion`;
+
+    const accordionItems = [...this.getAllAccordionItems()];
+
+    if (!accordionItems || accordionItems.length === 0) {
+      throw new Error(`No axa-accordion-item found inside ${this}!`);
+    }
+
+    accordionItems.forEach((item) => {
+      on(item, 'accordion-item-opened', (event) => {
+        this.accordionItemToggled(event);
+      });
+    });
+  }
+
+  disconnectedCallback() {
+    super.disconnectedCallback();
+  }
+
+  accordionItemToggled(event) {
+    event.stopPropagation();
```

Comment:
mmm... stopPropagation can cause problems for future componentens that may have to listen as well... are we sure is needed here?